bindata/cronjob-scheduler/config/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
I think we want all these files in this dir in the bindata/network/multus folder. Put them all on the same folder depth there and just number them.